mne/time_frequency/tfr.py

```diff
@@ -1151,7 +1151,7 @@ def plot(self, picks, baseline=None, mode='mean', tmin=None, tmax=None,
                                         mode=mode, layout=layout)
             _imshow_tfr(ax, 0, tmin, tmax, vmin, vmax, onselect_callback,
                         ylim=None, tfr=data[idx: idx + 1], freq=freqs,
-                        x_label='Time (ms)', y_label='Frequency (Hz)',
+                        x_label='t (s)', y_label='f (Hz)',
```

Comment:
t and f don't seem explicit enough for me

Reply:
When someone computes TFRs it should be clear what t (s) and f (Hz) mean. I prefer shorter labels to de-clutter the figures. But of course labels can be changed with MPL afterwards, although this is more work. Should I change it back?

Reply:
I don't think one-character labels are a good idea. You know your plot is supposed to be a TFR map, but when you're showing it to somebody else, the label is what tells them it's a TFR plot.